src/pkg/caendr/caendr/models/datastore/entity.py

```diff
@@ -310,6 +316,36 @@ def modified_on(self):
 
 
 
+  def _get_raw_prop(self, key, fallback=None):
+    return self.__dict__.get(key, fallback)
+
+  def _set_raw_prop(self, key, val):
+    self.__dict__[key] = val
+
+
+  def _get_enum_prop(self, EnumType, key, fallback=None):
```

Comment:
Implements & standardizes Entities having `Enum` props.  I'd like to redesign the entity property system entirely (like we've discussed in-person), but I think that's a slightly larger project, so for now I'm using these methods.